scripts/debugger-build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# Build the node in --release mode but with debug symbols to allow debugger breakpoints to work.
+
+RUSTFLAGS=-g cargo build --release
```

Comment:
Why not put it in a launch.json insteadd of a script ?

Reply:
Just found out about VSCode tasks.json, which seems to also be callable before a launch.json entry !